distributed/worker.py

```diff
@@ -1440,15 +1439,12 @@ async def start_unsafe(self):
         self.start_periodic_callbacks()
         return self
 
-    async def terminate(self, **kwargs):
-        return await self.close(nanny=True, **kwargs)
-
     @log_errors
     async def close(
         self,
         timeout=30,
         executor_wait=True,
-        nanny=False,
+        nanny=True,
```

Comment:
By changing the default back to True, resolves:
* https://github.com/dask/distributed/pull/6363/files#r878487787
* https://github.com/dask/distributed/pull/6363/files#r878491269
* https://github.com/dask/distributed/pull/6363/files#r878480001
* https://github.com/dask/distributed/pull/6363/files#r878494201